### PR TITLE
Update cloud-storage.rst

### PR DESCRIPTION
**Pull request by patrickgardella**
Microsoft released a new version with breaking changes in late 2019. Rasa uses the legacy version, so we need to tell users how to access the legacy version. Until the persistor is updated to the new version, of course.

**Proposed changes**:
- Update documentation to note use of legacy azure-storage-blob library.

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ X ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @wochinge will take a look at it as soon as possible ✨

File: docs/user-guide/cloud-storage.rst [hunk 1]
```diff
@@ -43,8 +43,8 @@ Rasa supports using `S3 <https://aws.amazon.com/s3/>`_ ,
     variable to the path of that key file.
 
 * Azure Storage
-    Azure is supported using the ``azure-storage-blob`` package,
-    which you can install with ``pip install azure-storage-blob``.
+    Azure is supported using the legacy ``azure-storage-blob`` package (v 2.1.0),
+    which you can install with ``pip install -Iv azure-storage-blob==2.1.0``.
```
Comment (wochinge) [inline docs/user-guide/cloud-storage.rst:47]:
```suggestion
    which you can install with ``pip install -I azure-storage-blob==2.1.0``.
```

**Comment by wochinge** [comment]
I also created an issue for updating the code: https://github.com/RasaHQ/rasa/issues/5736 

Would you like to work on that issue @patrickgardella ? I can support you and would be available for questions / feedback.

**Comment by wochinge** [comment]
@patrickgardella How is it going? Do you have any open questions about my comments?

**Pull request merged**